PostFreeSurfer/PostFreeSurferPipeline.sh

```diff
@@ -230,4 +283,8 @@ argList+="$CorrectionSigma "
 argList+="$RegName "                                  # ${39}
 "$PipelineScripts"/CreateMyelinMaps.sh ${argList}
 
+verbose_green_echo "---> Finished Post FreeSurfer Pipeline"
```

Comment:
This is a quibble, but IMO we should refer to the scripts by their actual names (without spaces) -- when you start adding spaces, it can get arbitrary where spaces should go for some of the more convoluted script names.
i.e.,
`verbose_green_echo "---> Finished PostFreeSurferPipeline"`

Reply:
Or in this script, where `script_name` is defined, it could simply be:
`verbose_green_echo "---> Finished ${script_name}"`

Reply:
Took out the spaces.

Reply:
Changed to using variable.